controllers/home.js

```diff
@@ -13,6 +13,15 @@ async function home(req, res) {
     csrfToken: req.csrfToken(),
   };
 
+  const coinFlipNumber = Math.random() * 100;
+  const experimentBranch = getExperimentBranch(req, coinFlipNumber);
+
+  const isUserInExperiment = isExperiment(experimentBranch);
```

Comment:
Since we're only calling `isExperiment` once, I think this could be 
`const isUserInExperiment = (experimentBranch === "vb" || experimentBranch === "vc")`

Reply:
Revised!